instrumentation/opentelemetry-instrumentation-httpx/tests/test_httpx_integration.py

```diff
@@ -763,3 +764,14 @@ def test_basic_multiple(self):
         self.perform_request(self.URL, client=self.client)
         self.perform_request(self.URL, client=self.client2)
         self.assert_span(num_spans=2)
+
+    def test_no_op_tracer_provider(self):
+        HTTPXClientInstrumentor().uninstrument()
+        HTTPXClientInstrumentor().instrument(
+            tracer_provider=trace_api.NoOpTracerProvider()
+        )
+         async with httpx.AsyncClient() as client:
```

Comment:
I think if you write test under the async test class, you should have test under sync as well. This will ensure that the instrumentation behaves correctly in both modes of the HTTPX client.

Reply:
Thank you for your comments. I am working on your recommendations and will be submitting the PR again.

Reply:
@zhihali - I have reviewed the code and don't think the test should be under sync as well. Can you please let me know why you think the test should be under sync too.